src/components/Footer.messages.js

```diff
@@ -73,9 +78,14 @@ const messages = defineMessages({
   },
   'footer.legalLinks.termsOfService': {
     id: 'footer.legalLinks.termsOfService',
-    defaultMessage: 'Terms of Service & Honor Code',
+    defaultMessage: 'Terms of Service',
     description: 'The label for the link to the edX terms of service page.',
   },
+  'footer.legalLinks.honorCode': {
+    id: 'footer.legalLinks.honorCode',
+    defaultMessage: 'Honor Code',
+    description: 'The label for the link to the edX honor code page.',
```

Comment:
```suggestion
    description: 'The label for the link to the honor code page.',
```

And if you don't mind applying this to the other messages, it would be great. :)

Reply:
Just to confirm, Should I apply this change to all the existing messages?

Reply:
It would be awesome if you did, yes.  Not a requirement, though.

Reply:
I have changed a few. Let me know if you want me to change other messages like the social media app messages.